Gemfile

```diff
@@ -65,6 +65,7 @@ gem 'arclight', github: 'sul-dlss/arclight'
 gem 'blacklight_range_limit', '~> 7.1'
 group :development, :test do
   gem 'solr_wrapper', '>= 0.3'
+  gem 'nulldb'
```

Comment:
does this need to be listed seperately or does activerecord-nulldb-adapter pull it in?